MVC/application/controller/afterLogin.php

```diff
@@ -46,8 +46,45 @@ public function userProfile() {
       else {
         session_destroy();
         echo "<script>alert('First log-in!!!');</script>";
-        //header("location: /userControl");
-        $this->view("userControl");
+        header("location: /userControl");
+        //$this->view("userControl");
+      }
+    }
+
+    public function postData() {
```

Comment:
Please add comments to define the purpose of the functions

Reply:
I will add comments later.